app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.length)
+  end
```

Comment:
- [ ] Optional suggestion

I would recommend doing something like this to count the likes.

```
def   update_likes_counter
  post.increment!(:likes_counter)
end
```